gen/definitions/aaa_accounting.yaml

```diff
@@ -74,3 +63,203 @@ attributes:
     xpath: system/guarantee-first
     tf_name: system_guarantee_first
     example: false
+  - yang_name: commands
+    type: List
+    exclude_test: true
+    attributes:
+      - yang_name: level
+        example: 15
+        id: true
+      - yang_name: list-name
+        example: COMMAND_LIST
+        id: true
+      - yang_name: commands-config/commands-choice/others-case/action-type
+        xpath: commands-config/action-type
```

Comment:
```
tf_name: action_type
```

...similarly for the attributes below, I believe we can remove the "commands_config_" prefix as it is already under a "commands" parent element